File: DAL/Models/City.cs
using NetTopologySuite.Geometries;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL.Models
{
    public class City
    {
        public int Id { get; set; }
        public string Name { get; set; }

        public Point Location { get; set; }
        //Location = new Point(16.3738, 48.2082)
        //{
        //    SRID = 4326
        //}

        public Country Country { get; set; }
        public ICollection<Company> Companies { get; set; }
        public ICollection<Job> Jobs { get; set; }
        public ICollection<Employee> Employees { get; set; }
    }
}

File: GlassDoor/ClientApp/src/app/models/city.ts
import { Company } from "./company";
import { Country } from "./country";
import { Employee } from "./employee";
import { Job } from "./job";

export class City {

  constructor(id?: number, name?: string, location?: string, country?: Country,
    company?: Company, job?: Job, employee?: Employee)
  {

    this.id = id;
    this.name = name;
    this.location = location;
    this.country = country;
    this.company = company;
    this.job = job;
    this.employee = employee;
  }
  public id: number;
  public name: string;
  public location: string;
  public country: Country;
  public company: Company;
  public job: Job;
  public employee: Employee;
}
